### ISSUE TITLE
Any E14 bulbs?

### ISSUE THREAD

**Issue by mcer12** [mariusmotea/diyHue#247]
OK so we have E27 covered by Yeelight (which works great by the way) but sadly they don't have an E14 model. Any tips on decent wifi E14 bulbs?

**Comment by mcer12** [comment]
@cheesemarathon  Sorry maybe I should've specified in the title I look for wifi bulbs. And also it should be full RGB. I so far found this:
https://www.aliexpress.com/item/Original-Xiaomi-Mijia-Smart-E14-LED-Bulb-White-and-Warm-Light-For-Xiaomi-Smart-Home-Kit/32853492483.html?spm=2114.search0104.0.0.55cf522dKOqbQY
but it's only adustable white.

**Comment by mariusmotea** [comment]
There are lot of E14, wifi and RGB bulbs on aliexpress, but in terms of size are almost like E24, only with different socket. Now the question is if they use esp8266 to flash them directly without any other changes.

**Comment by mcer12** [comment]
@mariusmotea Yes but they are very cheap and I wouldn't trust it in terms of safety and quality. I will take a more thorough look on the one in my previous link, see if there's an api for it. It's only white but it's small and it's philips.

**Comment by cheesemarathon** [comment]
I have yet to see any e14 bulbs that have been confirmed to use an Esp8266. So if you want quality and safety maybe stuck with e27 and get an e14 to e27 adapter? Or in some cases you may even be able to replace the socket completely.

**Comment by mcer12** [comment]
@cheesemarathon That sadly won't work because yeelight is huge (even for an e27) and heavy. Simply won't fit in the lamp...

**Comment by cheesemarathon** [comment]
Hi @mcer12 
Do you want to close this for now? Feel free to ask around some more on slack!
